packages/visx-brush/src/BaseBrush.tsx

```diff
@@ -320,6 +320,8 @@ export default class BaseBrush extends React.Component<BaseBrushProps, BaseBrush
     }
   };
 
+  debouncedHandleWindowPointerMove = debounce(this.handleWindowPointerMove, 1);
```

Comment:
could potentially add a comment for why we need this debounce since it was such a cryptic issue before!